src/diffusers/models/transformers/lavender_transformer_2d.py

```diff
@@ -0,0 +1,401 @@
+# Copyright 2024 Stability AI, Lavender Flow, The HuggingFace Team. All rights reserved.
```

Comment:
New model because it differs from the SD3 one:

* Uses register tokens
* Mixes MMDiT and another kind of simple DiT block (that uses a concatenated `encoder_hidden_states` and `hidden_states`) 
* The final layer norm is different 
* Position embeddings are different (uses learned positional embeddings) 
* The feedforward is different. We only support GeLU and its variants in the feedforward. It uses SwiGLU. 
* No pooled projections.